src/WarcraftLegacies.Source/Quests/BlackEmpire/QuestGiftofFlesh.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.QuestSystem;
+
+namespace WarcraftLegacies.Source.Quests.BlackEmpire
+{
+  /// <summary>
+  /// Get to Ranazjar to unlock the Aqir
+  /// </summary>
+  public sealed class QuestGiftofFlesh : QuestData
+  {
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestGiftofFlesh"/> class.
+    /// </summary>
+    public QuestGiftofFlesh() : base("Gift of Flesh",
+      "The original A'qir have always served me faithfully. They are probably buried deep beneath the earth. If I am to awaken them once more, I will need to search in the deepest reaches of Kalimdor: The Stonetalon caverns and the tunnels of Ahn'qiraj.",
```

Comment:
```suggestion
      "The original A'qir, now buried deep beneath the earth, have always served me faithfully. To awaken them once more, I must search in the deepest reaches of Kalimdor: the Stonetalon caverns and the tunnels of Ahn'qiraj.",
```
Just giving an air of certainty to the inner thoughts of this Old God.